### PR TITLE
fix: Metadata Not Being Saved

### PR DESCRIPTION
**Pull request by blessedcoolant**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Community Node Submission

## Have you discussed this change with the InvokeAI team?
- [ ] Yes
- [ ] No, because:

      
## Have you updated all relevant documentation?
- [ ] Yes
- [ ] No

## Description

Metadata was not getting saved coz the accumulator was not plugged in if watermark or nsfw nodes were turned off.

## Related Tickets & Documents

- Related Issue #
- Closes #

## Added/updated tests?

- [ ] Yes
- [ ] No : _please replace this line with details on why tests
      have not been included_

### CODE REVIEW

File: invokeai/frontend/web/src/features/nodes/util/graphBuilders/buildLinearTextToImageGraph.ts [hunk 1]
```diff
@@ -229,5 +229,21 @@ export const buildLinearTextToImageGraph = (
...
+      source: {
+        node_id: METADATA_ACCUMULATOR,
+        field: 'metadata',
+      },
+      destination: {
+        node_id: LATENTS_TO_IMAGE,
+        field: 'metadata',
+      },
+    });
+  }
```
Comment (maryhipp) [inline invokeai/frontend/web/src/features/nodes/util/graphBuilders/buildLinearTextToImageGraph.ts:246]: Really shouldn't this always be added, and the watermarker and nsfw functions _not_ add it?

Comment (maryhipp) [reply invokeai/frontend/web/src/features/nodes/util/graphBuilders/buildLinearTextToImageGraph.ts:246]: I guess its more complex than that though

Comment (blessedcoolant) [reply invokeai/frontend/web/src/features/nodes/util/graphBuilders/buildLinearTextToImageGraph.ts:246]: The watermarker and nsfw add it .. so I'm checking if those arent enabled. If they arent, then we add it. Maybe there's a better way of doing this.

**Review by psychedelicious — APPROVED** [review]

File: invokeai/frontend/web/src/features/nodes/util/graphBuilders/buildLinearTextToImageGraph.ts (hunk 1 above)
Comment (psychedelicious) [reply invokeai/frontend/web/src/features/nodes/util/graphBuilders/buildLinearTextToImageGraph.ts]: its fine to just write the metadata for latents to image, simpler for now, pr updated

**Pull request merged**